### ISSUE TITLE
Allow team screenshot to update team role

### ISSUE THREAD

**Issue by Kdog0073** [PokeNavBot/issue-tracker#304]
It was discovered in the code that an upcoming item to change teams is possible. While it originally made sense to only assign team roles once, this decision should be re-evaluated given this new item

**Comment by typographynerd** [comment]
I would second this idea and might suggest a step further, having a toggle function to disable changing team role by command. Our server admins like having to use a screenshot as it allows us a way to double-check the veracity of teams.

**Comment by dandesousa** [comment]
Is it enough to allow the team command / screenshot to be settable when a role is already set for a team?

How much fine grained control would you guys suggest?

**Comment by Kdog0073** [comment]
As of right now, everyone has access to the team command anyways, so control isn't the highest priority. So initially, just removing the restriction if a role is assigned is good enough.

In the most ideal case, Pokenav would have the user's ign and be able to scan the ign from the screenshot. If a player has an ign present, compare the ign to the screenshot. If not present, set their ign in their profile

**Comment by typographynerd** [comment]
Ultimately, what I was hoping for was to have the option to 1) have team profiles only set by screenshot and 2) only in one channel. However, of the two, I'd prefer 1 over 2—as I like being able to see that they have some evidence of the team they are claiming to be on.

I love the idea of grabbing the IGN from the screenshot and setting their server "nickname" to that.

**Comment by dandesousa** [comment]
I would split that into another ticket (reading in-game name) -- since it would change team screenshots to require OCR.

To summarize it sounds like we'd want a couple things:

1. Allow team screenshots even if a role is set.
2. Disallow use of the team command as a server option.

You should know though that theres only one profile entry for team in the database, so this would only impact the ability of the user to use team to change their team role on your server.

The other thing this makes confusing -- if screenshot scan fails, then admins will have to use there powers to give them the right role, and they will have to go into a DM and set their team manually.

I think this workflow is a little confusing but I'm fine with a flag that opts you into this.

**Comment by dandesousa** [comment]
This is blocked by #316 -- which should be done first
